src/components/collection/ItemBreadcrumb.tsx

```diff
@@ -0,0 +1,49 @@
+import { useRouter } from 'next/router';
+
+import React, { useContext } from 'react';
+
+import { Breadcrumbs, Button, Typography } from '@mui/material';
+
+import { ItemRecord } from '@graasp/sdk/dist/frontend/types';
+
+import { buildCollectionRoute } from '../../config/routes';
+import { QueryClientContext } from '../QueryClientContext';
+
+type ItemBreadcrumbProps = {
+  itemId: string;
+};
+
+const ItemBreadcrumb: React.FC<ItemBreadcrumbProps> = ({ itemId }) => {
+  const { hooks } = useContext(QueryClientContext);
+
+  const { data: item } = hooks.useItem(itemId);
+
+  const parents = hooks.useParents({
+    id: item?.id,
+    path: item?.path,
+    enabled: true,
+  });
```

Comment:
@clementcharmillot @pyphilia does this provide the full path to the topmost parent, or to the root published parent?

Reply:
@clementcharmillot to get the correct breadcrumbs the algorithm is the following:

1. Call `useItemTags` at the current item. This will return `{ tagId, itemId }`, find the tag with `tagId === PUBLISHED_TAG_ID`. The associated `itemId` will be the published parent.
2. Slice the head of the current item path until the index of the previous `itemId`
3. Split the resulting path and fetch all items in the remaining diff'd path